src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4960,6 +5006,14 @@ namespace AirflowNetworkBalanceManager {
 
         AllocateAirflowNetworkData();
 
+        bool OnOffFanFlag = false;
+        for (i = 1; i <= DisSysNumOfCVFs; i++) {
+            if (DisSysCompCVFData(i).FanTypeNum == FanType_SimpleOnOff) {
+                OnOffFanFlag = true;
+                break;
```

Comment:
With this `break` here, is there any chance something could fall through here where you have only initialized the first one?

Reply:
@Myoldmopar The break statement is removed.